src/driver-sinowealth.c

```diff
@@ -129,13 +131,14 @@ struct sinowealth_config_report {
 	 * 0x2 - 3 mm
 	 */
 	uint8_t lift_off_distance;
```

Comment:
These bytes are always 0xFF for me. I'm not sure if it works for Glorious mice, but for G-Wolves Hati it works like in the snippet in it's add request thread.